openc3-cosmos-init/plugins/packages/openc3-tool-common/src/tools/admin/ClassificationBannerSettings.vue

```diff
@@ -274,6 +274,20 @@ export default {
     },
   },
   watch: {
+    displayTopBanner: function (val) {
+      if (val) {
+        this.topHeight = 20
+      } else {
+        this.topHeight = 0
+      }
+    },
+    displayBottomBanner: function (val) {
+      if (val) {
+        this.bottomHeight = 20
+      } else {
+        this.bottomHeight = 0
+      }
+    },
```

Comment:
Height can still be changed but this fixes the issue where you forget to set the height and it's just 0 and doesn't appear. Basically I'm setting a sensible default.